.gitignore

```diff
@@ -12,3 +12,4 @@ phpstan.neon
 /.phpunit.result.cache
 /docs/_build/
 npm-debug.log
+/tests/App/public/bundles
```

Comment:
This require a dev-kit pr

Reply:
Same, already on dev kit https://github.com/sonata-project/dev-kit/blob/788cc95853a037eb1c4bcf51e66439b0b73b4f5d/templates/project/.gitignore.twig#L18

Reply:
I thought we had some `panther: true` to add. Forgot I made change about this.